lib/matchers/equals.js

```diff
@@ -0,0 +1,21 @@
+var _ = require('lodash');
+var factory  = require('../factory');
+
+module.exports = factory({
+  initialize: function(expectedValue) {
+    this.expectedValue = expectedValue;
+  },
+  match: function(path, value) {
+    if (typeof this.expectedValue === 'undefined') {
+      return 'should be defined';
+    }
+
+    if (_.isObject(this.expectedValue)) {
+      return 'should be a primitive';
```

Comment:
Throws an exception in the initialize time would be a better choice, because it means that `your spec definition is not correct` rather than `hey, the value should be primitive`.